### PR TITLE
Add missing state targets

### PR DESCRIPTION
**Pull request by OdeclasV**
- OH
- WA

### CODE REVIEW

**Comment by tomdionesotes** [comment]
Only see OH folks added here, no WA.

**Comment by OdeclasV** [comment]
Yeah, working on those right now. But wanted to commit and open the PR to keep things organized. I'll ping you again when WA folks are added @tomdionesotes

**Comment by OdeclasV** [comment]
WA folks added

File: members/WAL000166.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    - wait:
+        - value: 2
+    - click_on:
+        - value: Send
+          selector: "#FormSubmitButton"
+  success:
+    headers:
+      status: 200
+    body:
+      contains: The following message was sent to
```
Comment (tomdionesotes) [inline members/WAL000166.yaml]: You're sure this is the success message for all members?

Comment (OdeclasV) [reply members/WAL000166.yaml]: Si. I tested each one individually in the Tester and no failures. All of the success pages look like the one below 

image

**Pull request merged**